Makefile

```diff
@@ -128,7 +128,8 @@ $(DOCKER_BUILD_TARGETS):
 	$(eval DOCKER_BUILD_CMD := $(subst docker_build_,,$@))
 	docker build \
 		--build-arg VCS_REF=$(GIT_COMMIT) \
-		-t $(REGISTRY)/$(APP_NAME)-$(DOCKER_BUILD_CMD):$(BUILD_TAG) \
+		--build-arg ARCH=$(GOARCH) \
+		-t $(REGISTRY)/$(APP_NAME)-$(DOCKER_BUILD_CMD)-$(GOARCH):$(BUILD_TAG) \
```

Comment:
I think we may also want to include GOOS in the image tag here too? Else we will not be able to support amd64-windows in future (see #377)

Reply:
ping on this 😄